### ISSUE TITLE
All pictures have object. Why does "count" equate to 0?

### ISSUE THREAD

**Issue by baolinhu** [AlexeyAB/darknet#1016]
All pictures have object. Why does "count" equate to 0?
![image](https://user-images.githubusercontent.com/16516384/41191778-dacbfb9a-6c26-11e8-8638-c18bda2dabac.png)

**Comment by AlexeyAB** [comment]
Because for this Region (for these anchors that specified in the masks= for this [yolo]-layer) there is no corresponding object.

**Comment by baolinhu** [comment]
Do you think my training log is normal?thanks. @AlexeyAB 
![image](https://user-images.githubusercontent.com/16516384/41192641-7feefb0c-6c33-11e8-941a-43cbc18d85eb.png)

**Comment by AlexeyAB** [comment]
@baolinhu Yes. But learning rate too small.

**Comment by baolinhu** [comment]
This is in the final of train.But the loss is a little big.And the "obj" is not close to 1.

**Comment by AlexeyAB** [comment]
@baolinhu Set higher learning_rate= and higher steps=

**Comment by baolinhu** [comment]
I have tried.You can see in  ![issues1014](https://github.com/AlexeyAB/darknet/issues/1014#issuecomment-396016994)
